.apolo/src/apolo_apps_postgresql/outputs_processor.py

```diff
@@ -43,15 +80,53 @@ def _b64decode(s: T) -> T:
         return base64.b64decode(s).decode()
 
     user = _b64decode(secret_data["user"])
-    password = _b64decode(secret_data["password"])
+    password_value = _b64decode(secret_data["password"])
     host = _b64decode(secret_data["host"])
     port = _b64decode(secret_data["port"])
     pgbouncer_host = _b64decode(secret_data["pgbouncer-host"])
     pgbouncer_port = _b64decode(secret_data["pgbouncer-port"])
-    dbname = _b64decode(secret_data.get("dbname"))
+    original_dbname = _b64decode(secret_data.get("dbname"))
+
+    # Use override database or the original
+    dbname = database_override if database_override is not None else original_dbname
+    # Include database in secret key if it's an override
+    db_suffix = f"-{dbname}" if database_override else ""
+
+    postgres_conn_string = f"postgresql://{user}:{password_value}@{pgbouncer_host}:{pgbouncer_port}/{dbname}"
 
-    postgres_conn_string = (
-        f"postgresql://{user}:{password}@{pgbouncer_host}:{pgbouncer_port}/{dbname}"
+    # Create Apolo secrets for all sensitive fields
+    password = await create_apolo_secret_with_retry(
```

Comment:
what if we are updating app and the secret with the same name already exists? can we patch `create_apolo_secret` to recreate it?